lib/Makefile.am

```diff
@@ -127,6 +127,7 @@ pkginclude_HEADERS			+= \
 	lib/gsocket.h			\
 	lib/hostname.h			\
 	lib/host-resolve.h		\
+	lib/list-adt.h \
```

Comment:
unfortunately it's counterpart has disappeared from CMakeLists.txt during the latest rebase :\

Reply:
We'll send a PR :)